sklego/preprocessing/pandastransformers.py

```diff
@@ -173,12 +222,18 @@ def _check_column_names(self, X):
 
 
 class PandasTypeSelector(BaseEstimator, TransformerMixin):
-    """The `PandasTypeSelector` transformer allows to select columns in a pandas DataFrame based on their type.
+    """The `PandasTypeSelector` transformer allows to select columns in a DataFrame based on their type.
```

Comment:
Considering its name, we could do the following:

```py
class PandasTypeSelector(BaseEstimator, TransformerMixin):
    def __init__(self, include=None, exclude=None):
        warn(
            "Please use `TypeSelector` instead of `PandasTypeSelector`, `PandasTypeSelector` will be deprecated in future versions",
            DeprecationWarning,
        )
        return TypeSelector(include, exclude)
```
and then 

```py
class TypeSelector:
    ...
    !!! info "New in version 0.9.0"
```

Reply:
Yes we can rename it to have a more intuitive naming path, but as you spotted, it shouldn't matter too much as they are exported into `preprocessing`.